lib/UWS/models.py

```diff
@@ -213,6 +212,18 @@ def addResult(self, id=None, href=None, result=None):
             currResult = result(id=id, ref=currRef)
             self.results.append(currResult)
 
+    def _getOptional(self, parsed, elName):
+        """returns the text value of elName within the elementTree
+        parsed.
+
+        If elName doesn't exist, this returns None.
+        """
+        mat = parsed.find(elName, namespaces=parsed.nsmap)
```

Comment:
didn't know that namespaces are stored in `parsed.nsmap`.

NOTE TO SELF: change this everywhere and get rid of `UWSns`